site/docs/v1/tech/apis/_import-webauthn-request-body.adoc

```diff
@@ -0,0 +1,73 @@
+Indicates whether user verification was provided at the time the passkey was registered.
+
+[field]#credentials``[x]``.credentialId# [type]#[String]# [required]#Required#::
+The credential Id generated by the authenticator stored as a base64URL-encoded string. This value is used by authenticators to look up a passkey during an authentication ceremony.
+
+[field]#credentials``[x]``.data# [type]#[Object]# [optional]#Optional#::
+An object that can hold any information about the passkey that should be persisted.
+
+[field]#credentials``[x]``.discoverable# [type]#[Boolean]# [optional]#Optional# [default]#defaults to `false`#::
+Indicates whether the passkey is "client-side discoverable." Discoverable passkeys can be used in authentication ceremonies without first identifying the user (e.g. by requiring the user to complete a form field).
+
+[field]#credentials``[x]``.displayName# [type]#[String]# [required]#Required#::
+The display name for the passkey selected during registration.
+
+[field]#credentials``[x]``.id# [type]#[UUID]# [optional]#Optional# [default]#defaults to secure random UUID#::
+The Id of the passkey. If not specified a secure random UUID will be generated.
+
+[field]#credentials``[x]``.insertInstant# [type]#[Long]# [optional]#Optional# [default]#defaults to now#::
+The link:/docs/v1/tech/reference/data-types#instants[instant] when passkey was created.
+
+[field]#credentials``[x]``.lastUseInstant# [type]#[Long]# [optional]#Optional# [default]#defaults to now#::
+The link:/docs/v1/tech/reference/data-types#instants[instant] when passkey was last used.
+
+[field]#credentials``[x]``.name# [type]#[String]# [optional]#Optional# [default]#defaults to **[see description]**#::
+A unique name meant to disambiguate passkeys with the same [field]#credential.displayName#. If this value is omitted, a string of four alphabetic characters will be generated. The combination of characters will be unique among the passkeys in this bulk import request.
+
+[field]#credentials``[x]``.publicKey# [type]#[String]# [required]#Required#::
+The passkey's public key, encoded in PEM format.
+
+[field]#credentials``[x]``.relyingPartyId# [type]#[String]# [required]#Required#::
+The Relying Party Id used at the time the passkey was registered.
++
+[WARNING]
+====
+This value **must** match the `rpId` value when the passkey was created, _and_ the browser origin during WebAuthn authentication in FusionAuth must be compatible with that Relying Party Id.
+====
+
+[field]#credentials``[x]``.signCount# [type]#[Integer]# [optional]#Optional# [default]#defaults to `0`#::
+The last known signature count for the passkey.
+
+[field]#credentials``[x]``.transports# [type]#[List<String>]# [optional]#Optional# [default]#defaults to ``[ ]``#::
+A list of transport types supported by the authenticator that generated the passkey. This value is used as a hint during the WebAuthn authentication ceremony to help identify eligible authenticators.
```

Comment:
What does the default mean. `The default value indicates there are no supported transport types.` or something like that.

Reply:
The default is an empty list. Because this value is just a hint, that is fine. Until recently, Safari didn't provide the list of transports on the WebAuthn client API response.

Should I move the default to the description? Or I could leave the `[default]` as-is and add a message to the description that it can still find the authenticator without the transport hints?

Reply:
```suggestion
A list of transport types supported by the authenticator that generated the passkey. This value is used as a hint during the WebAuthn authentication ceremony to help identify eligible authenticators.
+
The default value is the empty set, which means that no hints are provided. This does not impact the authentication ceremony negatively.
```

Does that make sense? I think leaving the default as is but just explaining what it affects (not much, based on what you told me) would be helpful.